### ISSUE TITLE
No need to write prescription again

### ISSUE THREAD

**Issue by AnnieMungai** [opensrp/opensrp-client-giz-malawi#464]
No need for the pharmacist to write prescriptions again but rather the whole prescription from treatment should be carried forward to the pharmacy form. The pharmacist does not prescribe any treatment so everything that has been prescribed in the treatment tab should be carried forward and displayed on top of the pharmacy tab and the pharmacist should just carry out the instruction. 

The prescription is completed by the clinician and should be automatically copied into the "Pharmacy" form, once the clinician has saved the "Final Diagnosis and treatment" form and sync has taken place.

**Comment by AnnieMungai** [comment]
Multiple prescriptions will pick up the latest one filled. 
Partial forms table - save the latest partial form for the latest patient. Write code to get the values that were filled earlier based 
Based on the availability of the medication - the widget will auto-populate, the medication available question is also prepopulated. 
When multiple medications are given, and only some of them are available - 
Remove the prescribed medication available question - then manually remove the medication that is not available. 
Will need to amend the malaria medication queries. 
What happens to the non-issued prescriptions? Do we save them or leave them out completely.

**Comment by roywanyaga** [comment]
@AnnieMungai 
Please update the data-dictionary for the issue

**Comment by AnnieMungai** [comment]
@roywanyaga confirm issue 471 and 464 are one issue and will both take 3-5days.

**Comment by roywanyaga** [comment]
@AnnieMungai Yes, issue 471 and 464 are the same issue and will both take 5-7 days

**Comment by AnnieMungai** [comment]
@dubdabasoduba did we get a way to implement this?

**Comment by dubdabasoduba** [comment]
> @dubdabasoduba did we get a way to implement this?
https://github.com/opensrp/opensrp-client-giz-malawi/issues/471#issuecomment-993388969 this is approach will address this.
